### PR TITLE
Add import order configuration for Eclipse

### PR DESCRIPTION
**Pull request by apeteri**
Related to MNGSITE-465; the configuration matches the screenshot seen in https://github.com/apache/maven-site/pull/269#issuecomment-945622992 .

### CODE REVIEW

**Comment by slawekjaranowski** [comment]
Where is reference to this file and instruction how to use?

ASF license should be placed in file header.

**Comment by apeteri** [comment]
License header and documentation snippets added.

File: content/resources/developers/maven-eclipse.importorder [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+#Organize Import Order
+#Mon Nov 29 10:45:56 CET 2021
+0=javax
+1=java
+2=
+3=\#
```
Comment (slawekjaranowski) [inline content/resources/developers/maven-eclipse.importorder:23]: The convention is:
```
import javax.*
blank line
import java.*
blank line
import all other imports
blank line
import static all other imports
```
does this configuration cover it?

Comment (apeteri) [reply content/resources/developers/maven-eclipse.importorder:23]: The order of import groups is the same; blank lines are controlled in a different section (Preferences &rarr; Java &rarr; Code Style &rarr; Formatter) that describes all Java formatting rules.

blank_lines

The screenshot shows the default profile which specifies a single blank line between import groups.

**Comment by slawekjaranowski** [comment]
Thanks.

Can you also look at: 
https://issues.apache.org/jira/browse/MNGSITE-469

**Pull request merged**